src/components/va-alert/va-alert.css

```
@@ -26,16 +26,21 @@ div.alert {
   box-sizing: border-box;
 }
 
-div.alert:not(.full-width):not(.bg-only) {
+div:not(.full-width) div.alert:not(.bg-only) {
   border-left-style: solid;
   border-left-width: 10px;
 }
-div.alert.full-width.info,
-div.alert.full-width.warning {
+div.full-width,
+div.full-width {
```

```suggestion
div.full-width {
```
Does this not limit the full-width banner to `info` and `warning` types now?

---

Ah, I need to make an additional CSS change. I had intended to move this down towards the bottom but didn't finish implementing it.

---

Here it is: https://github.com/department-of-veterans-affairs/component-library/pull/186/commits/d137e5fab64900ef37feca6d51a63fcd883436c7